docusaurus-docs/community/contribute/local-setup-ui.md

```diff
@@ -9,46 +9,62 @@ To get started with conda-store-ui development, there are a couple of options. T
+3. Run `docker compose --profile local-dev up --build` to start the entire development stack. This step may take a few minutes the first time you run it.
+4. Open your local browser and go to [http://localhost:8000](http://localhost:8000) to see conda-store-ui.
+5. You can then log in using the default username of `username` and default password of `password`.
+
+**Note:** Hot reloading is enabled, so when you make changes to source files (i.e., files under the conda-store-ui/src/ directory), your browser will reload and reflect the changes.
 
-## Without Docker
+## Advanced Option: Run the UI locally (with the rest of the stack in Docker)
 
-This setup still uses Docker for supporting services but runs conda-store-ui locally.
+For more advanced development on conda-store-ui, the first option might not be sufficient. If you need to work extensively in the UI codebase, then you will probably want to run the UI web app locally rather than within a Docker container.
 
-### Set up your environment
+Note: this setup still uses Docker to run the rest of the Conda Store stack. That means that the Conda Store database, server, worker, and storage services will all run in Docker containers. But the frontend web app (conda-store-ui) will run locally (not in a Docker container) with this setup.
+
+### Set up your Conda environment
 
 This project uses [Conda](https://conda.io) for package management. To set up Conda, please see their [installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
-1. Change to the project root ` cd conda-store-ui`
-2. From the project root create the conda environment `conda env create -f environment_dev.yml`
-3. Activate the development environment `conda activate cs-ui-dev-env`
-4. Install yarn dependencies `yarn install`
 
-### Run the application
+1. Change to the project root: `cd conda-store-ui`
+2. From the project root, create the conda environment: `conda env create -f environment_dev.yml`
+
+### Run conda-store-ui locally (and the rest of the stack in Docker)
+
+Tip: Make sure you are in the project root: `cd conda-store-ui`
 
-1. Run `yarn run start` and wait for the application to finish starting up
-Open your local browser and go to [http://localhost:8000](http://localhost:8000) so see conda-store-ui.
-3. You can then log in using the default username of `username` and default password of `password`.
+1. Activate the development environment: `conda activate cs-ui-dev-env`
+2. Set environment variables. Copy `.env.example` to `.env`. All default settings should work, but if you want to test against a different version of conda-store-server, you can specify it in the `.env` file by setting the `CONDA_STORE_SERVER_VERSION` variable to the desired version.
+3. Set the package manager to match package.json (`packageManager`): `corepack enable`
```

Comment:
Where is this added/executed? Do I need to add 
"(`packageManager`): `corepack enable`" 
to a config somewhere?
Or do I just need to run `corepack enable` in the terminal? 

If its the latter, maybe explain that `corepack` is a required install. And also, seems like yarn is also a required install for the advanced approach?

Reply:
Does this clear things up?

```suggestion
3. Run: `corepack enable`. Corepack comes with Node.js which was installed when you set up your Conda dev environment. Running `corepack enable` ensures that the version of Yarn that you use in the next step matches the version required by the repo (see package.json > packageManager). Yarn also comes with Node.js but is not automatically added to your environment's path until you run `corepack enable`.
```

Reply:
Yep, makes sense! Thanks!